presto-docs/src/main/sphinx/release/release-0.129.rst

```diff
@@ -2,26 +2,34 @@
 Release 0.129
 =============
 
-Network Topology Aware Scheduling
----------------------------------
-
-The scheduler can now be configured to take network topology into account when
-scheduling splits. This is set using the ``node-scheduler.network-topology``
-config. See :doc:`/admin/tuning` for more information.
-
 General Changes
 ---------------
 
+* Fix a planner issue that could cause queries involving ``OUTER JOIN`` to
+  return incorrect results.
+* Some queries, particularly those using :func:`max_by` or :func:`min_by`, now
+  accurately reflect their true memory usage and thus appear to use more memory
+  than before.
+* Always clear screen when canceling query with ``ctrl-C``.
```

Comment:
There was only one entry, so it didn't seem worth having an entire section for it.